CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+## [UNRELEASED]
+### Added
+- Added a changelog
+
+### Changed
+- create orders without a contract_id
+- add get operation for orders
```

Comment:
I'd say if there was a way to _get operation for orders_ before and you're doing it differently now, then the behaviour changed. Otherwise it's new because - as KeepAChangelog states - it is a new feature. So I'm with @sfroehler